src/vm/gcenv.os.cpp

```diff
@@ -354,7 +354,16 @@ uint32_t GCToOSInterface::GetCurrentProcessCpuCount()
 {
     LIMITED_METHOD_CONTRACT;
 
+#ifndef FEATURE_PAL
     return ::GetCurrentProcessCpuCount();
+#else
+   size_t cpuLimit;
+
+   if (PAL_GetCpuLimit(&cpuLimit))
```

Comment:
Do you still need to do this if `GetCurrentProcessCpuCount` also checks the CPU limit?